deploy/awstools/awstools.py

```diff
@@ -20,6 +24,7 @@
 # this needs to be updated whenever the FPGA Dev AMI changes
 # You can find this by going to the AMI tab under EC2 and searching for public images:
 # https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Images:visibility=public-images;search=FPGA%20Developer;sort=name
+# And whenever this changes, you also need to update deploy/tests/test_amis_snake.json
```

Comment:
It might be possible to change `tests/conftest.py` to automagically generate the json so that it will have the correct ami name string and doesn't have to be manually updated.  I stopped short of doing that in this PR.